packages/mdc-textfield/icon/_variables.scss

```diff
@@ -22,5 +22,4 @@
 
 $mdc-text-field-icon-position: 12px !default;
 $mdc-text-field-icon-padding: 48px !default;
-$mdc-text-field-dense-icon-padding: 44px !default;
```

Comment:
I think, they way `!default` works in sass it assigns a value only if it is not already initialized. So the currently value of `$mdc-text-field-dense-icon-padding` is `44px` and the next assignment with value `38px` is ignored.

We need to remove second assignment to keep current behaviour intact.

As @williamernest mentioned in his previous comment can you please add this fix to select component for consistency?

Thanks!

Reply:
I reverted the changes to keep `dense-icon-padding` at 44px.